src/pages/EP_ManageEPTemplate.page

```diff
@@ -35,21 +35,37 @@
             <form class="slds-form slds-form--horizontal">
                 <apex:outputPanel styleClass="slds-grid" id="templateForm">
                     <div class="slds-col slds-col--padded slds-p-top--large slds-size--1-of-2">
-                        <div class="slds-form-element slds-p-top--small slds-is-required">
+                        <div class="slds-form-element slds-p-top--small">
                             <apex:outputLabel value="{!$ObjectType.Engagement_Plan_Template__c.Fields.Name.Label}" for="idName" styleClass="slds-form-element__label" />
                             <div class="slds-form-element__control">
                                 <apex:inputField id="idName" styleClass="slds-input" value="{!template.Name}" required="true"/>
                             </div>
                         </div>
-                        <div class="slds-form-element slds-p-top--small slds-is-required">
+                        <div class="slds-form-element slds-p-top--small">
                             <apex:outputLabel value="{!$ObjectType.Engagement_Plan_Template__c.Fields.Description__c.Label}" for="idDesc" styleClass="slds-form-element__label" />
                             <div class="slds-form-element__control">
                                 <apex:inputField id="idDesc" styleClass="slds-input" value="{!template.Description__c}"/>
                             </div>
                         </div>
+                        <!--template field set fields-->
+                        <apex:repeat value="{!$ObjectType.Engagement_Plan_Template__c.FieldSets.ManageEPTemplateFS}" var="field">
+                            <div class="slds-form-element slds-p-top--small">
+                                <apex:outputLabel value="{!field.Label}" for="idtest" styleClass="slds-form-element__label" />
+                                <div class="slds-form-element__control">
+                                    <apex:inputField id="idtest" value="{!template[field.FieldPath]}" 
+                                        styleClass="{!CASE(field.Type, 
+                                                'reference', 'lookupInput203 ' + field.FieldPath, 
+                                                'picklist', 'slds-select',
+                                                'multipicklist', 'slds-select', 
+                                                'boolean', 'slds-checkbox',
+                                                'textarea', 'slds-textarea',
+                                                'slds-input')}" />
+                                </div>
+                            </div>
+                        </apex:repeat>
```

Comment:
in my use of field sets, I've typically seen if the field is required in either the DB or just in the field set (using isRequired, isDBRequired), and then added a '*' and appropriate markup to make required.  Now I realize in this pull request you are removing some of your required's, I assume to support the delete button.  But even if we don't have SLDS or visualforce enforce the requirement, should we still include the red '*' so the user believes it is required, and that our controller's save validation will check?  You can see an example of this here: https://github.com/SalesforceFoundation/Cumulus/blob/dev/src/pages/LVL_LevelEdit.page#L142